config/vufind/contentsecuritypolicy.ini

```
@@ -44,11 +44,16 @@ script-src[] = "https:"
 connect-src[] = "'self'"
 ; If you are using Google Analytics, uncomment the line below
 ;connect-src[] = "https://*.google-analytics.com"
+; If you are using Matomo, fix the URL for your installation and uncomment the line below
+;connect-src[] = "http://server.address/matomo/matomo.php"
```

Maybe best to use https in the example, since that is the more likely scenario:

```suggestion
;connect-src[] = "https://server.address/matomo/matomo.php"
```

---

I agree, but then the example in config.ini (which I copied here) should be updated, too. And maybe many more http URLS in config/, too.

---

True. If you'd like to open a separate PR to reduce http:// in config.ini, please feel free -- and/or you can adjust the Matomo one here while following my other suggestion about adding more CSP notes to config.ini (which would seem appropriate in the scope of this existing PR).